lambeq/backend/drawing/drawing.py

```diff
@@ -124,7 +128,8 @@ def draw(diagram: Diagram, **params) -> None:
         backend = TikzBackend(
             use_tikzstyles=params.get('use_tikzstyles', None))
     else:
-        backend = MatBackend(figsize=params.get('figsize', None))
+        backend = MatBackend(figsize=params.get('figsize', None),
+                             wires_linewidth=params.get('wires_width', 1.25))
```

Comment:
@Ragunath1729 please be consistent with the param name. `wires_linewidth` should be fine for the `draw` param.

Should you also add the `box_linewidth` as param for this?

Reply:
Also, can you create constants in `drawing.py` for the numerical values we are using? For example:

```python
WIRES_WIDTH_MAT = 1.25
# etc.
```

Reply:
We don't have a requirement to change the box linewidth. its uniform across all diagrams and backends, right ?